src/routes/notifications/notifications.controller.spec.ts

```diff
@@ -237,4 +241,71 @@ describe('Notifications Controller (Unit)', () => {
         });
     });
   });
+
+  describe('DELETE /chains/:chainId/notifications/devices/:uuid/safes/:safeAddress', () => {
+    it('Success', async () => {
+      const uuid = faker.datatype.uuid();
+      const safeAddress = faker.finance.ethereumAddress();
+      const chain = chainBuilder().build();
+      const expectedProviderURL = `${chain.transactionService}/api/v1/notifications/devices/${uuid}/safes/${safeAddress}`;
+      mockNetworkService.get.mockImplementation((url) =>
+        url.includes(`${safeConfigUrl}/api/v1/chains/`)
+          ? Promise.resolve({ data: chain })
+          : rejectForUrl(url),
+      );
+      mockNetworkService.delete.mockImplementation((url) =>
+        url === expectedProviderURL ? Promise.resolve() : rejectForUrl(url),
+      );
+
+      await request(app.getHttpServer())
+        .delete(
+          `/chains/${faker.random.numeric()}/notifications/devices/${uuid}/safes/${safeAddress}`,
+        )
+        .expect(200)
+        .expect({});
+      expect(mockNetworkService.delete).toBeCalledTimes(1);
+      expect(mockNetworkService.delete).toBeCalledWith(expectedProviderURL);
+    });
+
+    it('Failure: Config API fails', async () => {
+      const uuid = faker.datatype.uuid();
+      const safeAddress = faker.finance.ethereumAddress();
+      mockNetworkService.get.mockImplementation((url) =>
+        url.includes(`${safeConfigUrl}/api/v1/chains/`)
+          ? Promise.reject(new Error())
+          : rejectForUrl(url),
+      );
+
+      await request(app.getHttpServer())
+        .delete(
+          `/chains/${faker.random.numeric()}/notifications/devices/${uuid}/safes/${safeAddress}`,
+        )
+        .expect(503);
+      expect(mockNetworkService.delete).toBeCalledTimes(0);
+    });
+
+    it('Failure: Transaction API fails', async () => {
+      const uuid = faker.datatype.uuid();
+      const safeAddress = faker.finance.ethereumAddress();
+      const chain = chainBuilder().build();
+      mockNetworkService.get.mockImplementation((url) =>
+        url.includes(`${safeConfigUrl}/api/v1/chains/`)
```

Comment:
Same as https://github.com/5afe/safe-client-gateway-nest/pull/268#discussion_r1102693845

Reply:
Changed in [20a87a3](https://github.com/5afe/safe-client-gateway-nest/pull/268/commits/20a87a357afa6c9e9059a2e259a0a64d3468d812)